TestForSignUp/ElementFiller.cs

```diff
@@ -0,0 +1,43 @@
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Data;
+using System.Text;
+using System.Threading;
+
+namespace TestForSignUp
+{
+    class ElementFiller
+    {
+        ReadOnlyCollection<IWebElement> inputs;
+        ReadOnlyCollection<IWebElement> inputsWithoutDate;
+        ReadOnlyCollection<IWebElement> emailInputs;
+        
+        internal ElementFiller(ChromeDriver browser)
```

Comment:
Why you need `internal` here and in many of other methods and fields?

Reply:
fixed